MSAL/src/MSALDeviceInformation.m

```diff
@@ -32,6 +32,7 @@
 #import "ASAuthorizationSingleSignOnProvider+MSIDExtensions.h"
 
 NSString *const MSAL_DEVICE_INFORMATION_SSO_EXTENSION_FULL_MODE_KEY = @"isSSOExtensionInFullMode";
+NSString *const MSAL_DEVICE_INFORMATION_MDM_ID_KEY = @"mdm_id";
```

Comment:
Would it be possible to use MSID_BROKER_MDM_ID_KEY over here to avoid same key defined in multiple places?

Reply:
done